ansible/templates/minigraph_png.j2

```diff
@@ -72,10 +72,11 @@
 {% endfor %}
 {% endfor %}
 {% endfor %}
+{% if switch_type is not defined or switch_type != "voq" %}
 {% for asic_intf in front_panel_asic_ifnames %}
       <DeviceLinkBase i:type="DeviceInterfaceLink">
         <ElementType>DeviceInterfaceLink</ElementType>
-        <Bandwidth>40000</Bandwidth>
+        <Bandwidth>{{ port_speed[port_alias[loop.index - 1]] }}</Bandwidth>
```

Comment:
This change is not applicable of VoQ chassis ?

Reply:
With regards to line 75, I added the if condition because not all our ports on the linecards are connected to a fanout (only a few ports). Without this check, we were adding a DeviceInterfaceLink for every front panel port. minigraph.py in sonic-buildimage would then admin enable all the ports. But, since not all the ports are cabled, they would not become operationally up, and thus failing pre-sanity check.

We probably need to look at conn graph to see if the port is defined, and if so then we add a DeviceInterfaceLink. Let me check that out.

Reply:
Pushed a commit to add the DeviceInterfaceLink if interface present in the device connection table.